### PR TITLE
Update configuration.md

### PR DESCRIPTION
**Pull request by yangwenmai**
-----
[View rendered docs/configuration.md](https://github.com/yangwenmai/weekly-digest/blob/patch-1/docs/configuration.md)

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -5,7 +5,8 @@ The app provides features to customize your application. To configure your appli
 2. Add the following configuration in the file:
    ```yaml
    # Configuration for weekly-digest - https://github.com/apps/weekly-digest
-   publishDay: sun
+   # publishDay value: 0, 1, 2, 3, 4, 5, 6
```
Comment (abhijeetps) [inline docs/configuration.md]: Hey @yangwenmai , thanks for the PR.
But I think I won't be accepting this PR as the previous _configuration.md_ file is more human readable.
Also, I have configured the bot to understand both numeric values of week or substring of the week.
So, using 
`publishDay: sun`
is same as using
`publishDay: 0`
for the app.
I will be closing this PR now. Thank you. 😃

Comment (yangwenmai) [reply docs/configuration.md]:
```js
config = fixConfig(config)
   console.log(`Publish Day: ${getNumDayFromLongDay(config.publishDay)}, Today: ${moment.utc().day()}`)
   if (moment.utc().day() === getNumDayFromLongDay(config.publishDay)) {
   await weeklyDigest(context, {owner, repo, headDate, tailDate}, config)
  }
```

It's just use number.

Comment (yangwenmai) [reply docs/configuration.md]: If u're set them in your bot, maybe u can list those human readable string.

Comment (abhijeetps) [reply docs/configuration.md]: See this: https://github.com/probot/weekly-digest/blob/master/src/markdown/getNumDayFromLongDay.js
Here, I have configured the bot to return a numerical value of the week, if the data is of string type.
And this is the _getNumDayFromLongDay_ method.

Comment (yangwenmai) [reply docs/configuration.md]: Sorry, my fault, and  I understand.
But u're example config is `sun`, not `sunday`.

So, maybe u can set all of the values.

**Pull request closed without merging**